Swift/QtUI/QtXMLConsoleWidget.h

```diff
@@ -42,5 +42,7 @@ namespace Swift {
         private:
             QTextEdit* textEdit;
             QCheckBox* enabled;
+            QCheckBox* debugenabled;
```

Comment:
Ideally all pointers are initialised with `nullptr` if you don't directly assign a correct value to them at declaration time.